basis/concurrency/distributed/distributed.factor

```diff
@@ -37,22 +59,46 @@ SYMBOL: local-node
 : start-node ( addrspec -- )
     <node-server> start-server local-node set-global ;
 
-TUPLE: remote-thread node id ;
+TUPLE: remote-thread node id connection ;
```

Comment:
I'm concerned that storing the connection here will cause it to be serialized and sent across the connection, see for example ``M\ thread (serialize)``.

Reply:
What would you suggest? A separate global assoc to link `remote-thread`s with their `connection`s?